seatunnel-api/src/main/java/org/apache/seatunnel/api/source/SeaTunnelRuntimeEnvironment.java

```diff
@@ -24,12 +24,7 @@
  */
 public interface SeaTunnelRuntimeEnvironment {
```

Comment:
`xxxAware` is a better class name when there is only one` setXXX `method.
Refer to` spring framewok` `org.springframework.web.context.ServletConfigAware ``org.springframework.context.ApplicationContextAware`

Reply:
This's good idea! Can you contribute to it?

Reply:
No problem